recipe/meta.yaml

```diff
@@ -109,41 +119,35 @@ outputs:
     {% set tests_to_skip = tests_to_skip + " or test_full_reimport" %}          # [ppc64le or s390x]
     {% set tests_to_skip = tests_to_skip + " or test_pep338" %}                 # [ppc64le or s390x]
     {% set tests_to_skip = tests_to_skip + " or test_no_typing_extensions" %}   # [ppc64le or s390x]
-    # 2022/5/5: E       AssertionError: (131, 1.54987431981169249551435343964035e-09, 4.0035173453529620614007210953362e-19, 'arcsinh')
-    # Flawed test when using MKL
-    # https://github.com/numpy/numpy/issues/16769
-    {% set tests_to_skip = tests_to_skip + " or test_overrides" %}  # [blas_impl == 'mkl']
-    # https://github.com/numpy/numpy/issues/15243
-    {% set tests_to_skip = tests_to_skip + " or test_loss_of_precision" %}  # [s390x or ppc64le]
-    # It should be fixed by https://github.com/numpy/numpy/issues/20426
-    # but it still fails on some platforms.
-    {% set tests_to_skip = tests_to_skip + " or test_new_policy" %}   # [s390x or ppc64le or (osx and arm64)]
-    # On osx-arm64: FAILED core/tests/test_limited_api.py::test_limited_api - subprocess.CalledProcessor
-    {% set tests_to_skip = tests_to_skip + " or test_limited_api" %}   # [osx and arm64]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_gcd_overflow" %}  # [s390x]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_herm_cases" %}  # [osx and arm64]
-    {% set tests_to_skip = tests_to_skip + " or test_generalized_herm_cases" %}  # [osx and arm64]
-    # ?
-    {% set tests_to_skip = tests_to_skip + " or test_partial_iteration_cleanup" %}  # [osx or win64]
+    # for version 1.23.2 of numpy the following tests must be skipped
+    {% set tests_to_skip = tests_to_skip + " or test_scalarmath" %}
+    {% set tests_to_skip = tests_to_skip + " or test_system_info" %}
+    {% set tests_to_skip = tests_to_skip + " or test_extending" %}
+    {% set tests_to_skip = tests_to_skip + " or test_typing" %}
+    {% set tests_to_skip = tests_to_skip + " or test_io" %}
+    {% set tests_to_skip = tests_to_skip + " or test_mem_policy" %}                    # [ppc64le or s390x]
+    {% set tests_to_skip = tests_to_skip + " or test_generalized_herm_cases" %}
+    {% set tests_to_skip = tests_to_skip + " or test_herm_cases " %}
```

Comment:
Having to disable this many tests, especially on x86_64, is an indication that something else is wrong with our build. The fix lies elsewhere; i.e., not in disabling tests.